dev-lang/nim/nim-1.6.10-r2.ebuild

```diff
@@ -105,8 +127,89 @@ src_compile() {
 
 src_test() {
 	local -x PATH="${S}/bin:${PATH}"
+	local skipfile="${T}/testament-skipfile.txt"
+	local nimflags=(
+		# Leave only the safe hints enabled
+		--hint:all:off
+		--hint:User:on
+		--hint:UserRaw:on
+	)
+	local testament_args=(
+		--skipFrom:"${skipfile}"
+		--nim:"bin/nim"
+		--targets:"$(usex test-js 'c js' 'c')"
+	)
 
-	edo ./koch test
+	[[ "${NOCOLOR}" == true || "${NOCOLOR}" == yes ]] && \
+		testament_args+=( --colors:off )
+
+	cat > "${skipfile}"  <<- EOF || die "Failed to disable tests"
```

Comment:
I wonder if we can have that in `${FILESDIR}`... wdyt?

Reply:
Yeah, the list is really long right now. Pushed